libc/include/llvm-libc-macros/stdlib-macros.h

```diff
@@ -17,6 +17,6 @@
 #define EXIT_SUCCESS 0
 #define EXIT_FAILURE 1
 
-#define RAND_MAX 32767
+#define RAND_MAX 2147483647
```

Comment:
Shall we define it as INT_MAX instead?

Reply:
I wasn't sure if that would always be available when this header is included.

Reply:
given that the implementation shifts right by 32, I think it's likely safe to have this be written out.